src/odemis/gui/model/__init__.py

```diff
@@ -1112,6 +1102,39 @@ def __init__(self, main):
         for rect in carrier_params["background"]:
             self.background.append((minx + rect[0], miny + rect[1], minx + rect[2], miny + rect[3]))
 
+        # Overview streams
+        self.overview_streams = model.VigilantAttribute({})  # dict: int --> stream or None
```

Comment:
Typically we avoid dicts in VAs, because it's "mutable". So it's possible to change its content, without the VA noticing. As typically the int are 1->9, I think you could do with a ListVA. (Lists are mutable too, but the ListVA uses extra tricks).

Reply:
The dict also has some advantages. It's definitely easier to index. If I want to insert the image for scinitillator 1, I just use .overview_streams[1]. For the ListVA, we need to index from 0. For now that's not such a big deal, but what if we have different scinitllator names in the future, maybe "A", "B", "C" instead of 1-9? For me the dict seemed less confusing, although I agree that it's annoying that we have to explicitly notify. If you insist, I'll change it.